daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -802,13 +834,22 @@ private[lf] final class Compiler(
   ): (t.SDefinitionRef, SDefinition) = {
     // Translates 'create Foo with <params>' into:
     // CreateDefRef(tmplId) = \ <tmplArg> <token> ->
-    //   let _ = $checkPrecond(tmplId)(<tmplArg> [tmpl.precond ++ [precond | precond <- tmpl.implements]]
+    //   let _ = checkPreCondition(tmplId, <tmplArg>) [tmpl.precond ++ [precond | precond <- tmpl.implements]]
```

Comment:
Based not his comment, it currently looks like we only work with `tmpl.precond`? Is the additional interface based pre-condition checking something that is managed elsewhere or is it a known issue that has yet to be fixed? Asking as I'm concerned we might have missed something here.

Reply:
We should drop the comment. 
For now interfaces do not have precondition.